.azurePipeline/templatePublishLogsFromPods.yml

```diff
@@ -0,0 +1,30 @@
+# Template to login into Dockerhub, build a docker image and push it.
+
+# Parameters:
+# `logsFolderPath` is the folder where the logs from the pods will be stored.
+# `releaseName` is the name of the release used to get the relevant list of pods.
+# `kubeConfigFile` is the kubeConfigFile path.
+
+parameters:
+  logsFolderPath: ''
+  releaseName: ''
+  kubeConfigFile: ''
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '3.7'
+  condition: failed()
```

Comment:
I suggest we use Azure's variables feature here so we can decide in the ci/cd system to keep the logs without having to inject a test failure.

https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch

Reply:
I do not see the point of using variables for this. In my mind, there three options:
 - we should publish the logs for every build (changing the condition from `failed()` to `always()`
 - we should never publish the logs (which I would assume is not preferred, otherwise we wouldn't have opened the issue #418 )
 - the logs are useful in some cases.
I went with the third option as I do not think that they are useful when a build succeeds (but I'm not feeling to strongly about it).
However, adding a variable `true` or `false` is in my opinion a bad idea, as I do not believe a developer should have to modify the CI script on a per branch basis.

Reply:
Compromise: the logs are always provided.